submissions/kolawoleOgunfowokan/HTML/index.html

```diff
@@ -1,42 +1,207 @@
-<!Doctype html>
+<!DOCTYPE html>
 <html lang="en">
-    <head>
-        <title>myoja-Login Page</title>
-        <meta charset="utf-8">
-        <meta name="viewport" content="width=device-width, initial-scale=1.0">
-        <link rel="stylesheet" href="./css/style.css">
-    </head>
-    <body>
-        <section class="login-page-head">
-            <img src="./images/diamond3.jpg" alt="logo" class="logo logo-image">
-            <span class="logo logo-text">myoja.com</span>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="./css/main-style.css">
+    <title>myoja.com</title>
+</head>
+<body>
+    <header class="page-header">
+        <div class="nav-head">
+            <img src="./images/diamond-simple.png" alt="logo" class="logo">
+            <p class="logo-text">myoja.com</p>
+            <div class="njffsaad">
+                <ul class="form-page">
+                    <li class="nav-head-list"><input type="search" name="search" class="search" placeholder="search for products, brand and categories"></li>
```

Comment:
This is good but can be better by indenting the elements properly to make it more readable. 
this for example: ```
<li class="nav-head-list">
        <input type="search" name="search" class="search">
</li>
```
looks much better than what you have currently. what do you think?

Reply:
Same applies to other places in your HTML files where you have nested HTML elements on a single line.

Reply:
I have removed spaces where necessary and also did proper indenting, for nested elements.